cmd/oceantv/broadcast_hardware_machine.go

```diff
@@ -247,6 +259,20 @@ func (sm *hardwareStateMachine) handleHardwareResetRequestEvent(event hardwareRe
 	}
 }
 
+func (sm *hardwareStateMachine) handleControllerFailureEvent(event controllerFailureEvent) {
+	sm.log("handling controller failure event")
+	switch sm.currentState.(type) {
+	case *hardwareOn, *hardwareRestarting, *hardwareStarting:
+		sm.transition(newHardwareStopping(sm.ctx))
+	case *hardwareOff:
+		sm.transition(newHardwareStarting(sm.ctx))
```

Comment:
If the hardware is in an off state, and we get a controller failure event, do we want to try transitioning to a hardware starting state ?

Reply:
Hmmm... This must have come from where I copied this layout from. Definitely not I wouldn't think

Reply:
Done.